cfgov/unprocessed/js/organisms/Summary.js

```diff
@@ -39,6 +39,13 @@ function Summary( element ) {
       return this;
     }
 
+    /* Bail out of initializatiion if the height of the summary's content
+       is less then our summary height of 5.5ems (16 * 5.5 = 88) */
```

Comment:
Maybe include a URL reference to where this is defined in Less?

Reply:
Added in https://github.com/cfpb/consumerfinance.gov/pull/6252/commits/9424b2ee90c6b75dfb1ded28ac2c343f724a8a7d

Also added `u-hidden` on the buttons by default to hide the flash on load you mentioned above.